opentelemetry-stdout/examples/basic.rs

```diff
@@ -0,0 +1,52 @@
+//! run with `$ cargo run --example basic --all-features
```

Comment:
Nit: Can we put this example with the other examples in the top level folder?

Reply:
I think we're trying to pare back the amount of examples we have in the top level, if anything we could remove this example.